app/cdap/components/RecipeManagement/RecipeDetails/DrawerHeader/index.tsx

```diff
@@ -0,0 +1,148 @@
+}
+
+const MainHeadWrapper = styled(Box)`
+  margin-top: 20px;
+`;
+
+const HeadWrapper = styled(Box)`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const RecipeHeaderLabel = styled(Typography)`
+  font-weight: 400;
+  font-size: 20px;
+  color: ${grey[900]};
+  padding: 1px;
+`;
+
+const IconsWrapper = styled(Box)`
+  display: flex;
+  align-items: center;
+`;
+
+const StyledCommonIconButton = styled(IconButton)`
+  padding: 12px;
+`;
+
+const CommonIcons = css`
+  fill: ${grey[600]};
+  width: 25px;
+  height: 25px;
+`;
+
+const StyledCloseIcon = styled(CloseIcon)`
+  ${CommonIcons}
+`;
+
+const VerticalDivider = styled(Box)`
+  width: 1px;
+  height: 25px;
+  border-left: 1px solid ${grey[300]};
+  margin: 0 5px;
+`;
+
+const HeaderWithUnderline = styled(Box)`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledActionButton = styled(Button)`
+  color: ${grey[600]};
+  font-weight: 500;
+  font-size: 14px;
+  letter-spacing: 1.25px;
+`;
+
+const StyledPopover = styled(Popover)`
+  & .MuiPopover-paper {
+    width: 164px;
+  }
+`;
+
+const StyledMenuItem = styled(MenuItem)`
+  padding: 12px 16px;
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 0.5px;
+  color: ${grey[900]};
+`;
+
+const PREFIX = 'features.WranglerNewUI.RecipeDetails';
+
+export default function DrawerHeader({ onCloseDetail }: IDrawerHeaderProps) {
+  const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
+  const open = Boolean(anchorEl);
+  const id = open ? 'simple-popover' : undefined;
+
+  const handleActions = (event: MouseEvent<HTMLButtonElement>) => setAnchorEl(event.currentTarget);
+  const handleClose = () => setAnchorEl(null);
+
+  return (
+    <MainHeadWrapper>
+      <HeadWrapper>
+        <HeaderWithUnderline>
+          <RecipeHeaderLabel>{T.translate(`${PREFIX}.drawerHeader`)}</RecipeHeaderLabel>
+        </HeaderWithUnderline>
+        <IconsWrapper>
+          <StyledActionButton
+            data-testid="actions-applied-button"
+            aria-describedby={id}
+            variant="text"
+            onClick={handleActions}
+          >
+            {T.translate(`${PREFIX}.actions`)}
+            {anchorEl && <ArrowDropUpIcon />}
+            {anchorEl === null && <ArrowDropDownIcon />}
+          </StyledActionButton>
+          <StyledPopover
+            id={id}
+            open={open}
+            anchorEl={anchorEl}
+            onClose={handleClose}
+            anchorOrigin={{
+              vertical: 'bottom',
+              horizontal: 'left',
+            }}
+          >
+            <StyledMenuItem>{T.translate(`${PREFIX}.edit`)}</StyledMenuItem>
+            <StyledMenuItem>{T.translate(`${PREFIX}.download`)}</StyledMenuItem>
+            <StyledMenuItem>{T.translate(`${PREFIX}.delete`)}</StyledMenuItem>
+          </StyledPopover>
+          <VerticalDivider />
+          <StyledCommonIconButton data-testid="close-detail-icon-button" onClick={onCloseDetail}>
```

Comment:
any specific reason for giving common word in StyledCommonIconButton check once if we can give it as CloseButton